samples/tab-approval/nodejs/Manifest/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<MicrosoftAppId>>",
+  "packageName": "com.microsoft.teams.tabrequestapproval",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.teams.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Tab Approval",
+    "full": "Activity feed notification to approve/reject user's request."
+  },
+  "description": {
+    "short": "App helps to notifiy manager regarding pending requests raised by user.",
+    "full": "This is an Tab app which send notification to manager for pending requset send by user."
```

Comment:
"This is an Tab app where requester can raise a request and a notification send to manager for its approval or rejection.